src/scripts/crowdin-import.ts

```diff
@@ -0,0 +1,420 @@
+// Library requires
+const i18Config = require("../../i18n.config.json")
+const {
+  copyFileSync,
+  existsSync,
+  mkdirSync,
+  readdirSync,
+  readFileSync,
+} = require("fs")
+const { resolve, join } = require("path")
+const argv = require("minimist")(process.argv.slice(2))
+
+/******************************
+ * Console flags              *
+ ******************************/
+
+/**
+ * -b,--buckets    Prints buckets overview and exits
+ * -v,--verbose    Prints verbose console logs
+ * -f,--full       Prints full name of buckets in summary
+ */
+
+/******************************
+ * Instructions for use       *
+ ******************************/
+
+/**
+ * 1. Run `yarn crowdin-clean` to initialize fresh ./.crowdin folder. This can also be used to erase contents when finished.
+ *
+ * 2a. Export/import CSV of languages ready for review:
+ *   1. Open "Website translation board" document in ethereum.org Notion (internal only)
+ *   2. Switch view of "Translation status by language" table to "All reviewed"
+ *   3. Click triple-dot (...) menu in TOP right corner of the entire app
+ *   4. Select "Export" > "Export as CSV"
+ *      Export format: Markdown & CSV
+ *      Include databases: Current view
+ *      Include content: No files or images
+ *      Include subpages: Off
+ *      Click "Export" > Save zip file
+ *   5. Unzip contents into (or copy into) ./.crowdin folder in the root of this repo
+ *
+ * 2b. Alternatively, you can manually add buckets to import to the USER_OVERRIDE object below.
+ *   1. Add the number of the corresponding content bucket to the chosen language array below
+ *      ie. `es: [1, 10],` would import the "Homepage" and "Learn" buckets for Spanish
+ *   2. Save file without committing*
+ *
+ * Optionally: To view summary of buckets from CSV, run `yarn crowdin-import --buckets` or `yarn crowdin-import -b`
+ *   Any items in USER_OVERRIDE will override the CSV import
+ *
+ * 3. Export translated content from Crowdin and import into ./.crowdin folder:
+ *   1. Export latest translated content from Crowdin and unzip
+ *   2. Copy languages folder from Crowdin export to ./.crowdin
+ *      ie. ./.crowdin/{lang-codes}
+ *
+ * 4. Execute script:
+ *   1. Execute script by running `yarn crowdin-import`
+ *   2. If successful, copy `GATSBY_BUILD_LANGS={langs}` output and paste in
+ *      your `.env`, then build site to test results.
+ *
+ * *Remember: Revert any working changes to this file before committing Crowdin import
+ */
+
+type BucketsList = { [key: string]: Array<number> }
+const USER_OVERRIDE: BucketsList = {
+  // FORMAT: lang_code: [bucket_number, bucket_number, ...],
+  // EXAMPLE: es: [1, 10, 12, 14],
+  ar: [2]
+
```

Comment:
I assume we don't actually want this committed... would leave this object blank, with only the commented-out example left behind for reference.

```suggestion
```